mount/losetup_linux.go

```diff
@@ -69,7 +69,10 @@ func getFreeLoopDev() (uint32, error) {
 	return uint32(num), nil
 }
 
-func setupLoopDev(backingFile, loopDev string, param LoopParams) error {
+// setupLoopDev attaches the backing file to the loop device and returns
+// the file handle fo the loop device. The caller is responsible for
```

Comment:
nit: typo (fo instead of for)

not a blocker, just in case other changes are needed before merge

Reply:
Ah right. Will fix.